### ISSUE TITLE
Uncaught Error: UNKNOWN, open 'C:\Users\user\.atom\packages\save-session\projects\undefined\project.json'

### ISSUE THREAD

**Issue by Digital-Thor** [mpeterson2/save-session#77]
I get this error intermittently after starting Atom.  At one point the `projects\undefined\project.json` file was up to 8 MB in size.  I uninstalled and re-installed the save-session package and things were fine for a while, but now the error is coming back intermittently.

**Steps to reproduce:**

1. Open Atom
2. Error may or may not occur
3. Not certain, but there may be some correlation regarding whether I was making changes in the 'settings' panel during a session prior to the session when the error occurs.

**Atom Version**: 0.186.0
**System**: Microsoft Windows 7 Professional 
**Thrown From**: Atom Core

### Stack Trace

Uncaught Error: UNKNOWN, open 'C:\Users\user\.atom\packages\save-session\projects\undefined\project.json'

```
At fs.js:82

Error: UNKNOWN, open 'C:\Users\user\.atom\packages\save-session\projects\undefined\project.json'
  at Error (native)

```

### Commands

```
```

### Config

```json
{
  "core": {
    "themes": [
      "atom-dark-ui",
      "atom-dark-syntax"
    ],
    "disabledPackages": [
      "language-objective-c",
      "language-c",
      "language-clojure",
      "language-csharp",
      "language-java",
      "language-perl",
      "language-php",
      "language-python",
      "language-ruby",
      "language-ruby-on-rails",
      "language-yaml",
      "language-go",
      "wrap-guide",
      "autocomplete-plus",
      "autocomplete-html"
    ],
    "projectHome": "E:\\Data\\Dropbox\\meteor\\examples",
    "ignoredNames": [
      ".*",
      "*~"
    ]
  },
  "editor": {
    "invisibles": {},
    "showIndentGuide": true
  }
}
```

### Installed Packages

```coffee
# User
atom-color-highlight, v3.0.8
color-picker, v1.4.4
csslint, v1.0.3
emmet, v2.3.4
file-types, v0.3.0
hide-files, v0.1.5
jshint, v1.3.0
meteor-api, v2.12.0
save-session, v0.11.5
show-in-folder, v0.1.0
windows-context-menu, v0.3.1

# Dev
No dev packages
```

**Comment by xdumaine** [comment]
This looks similar to the fix in #81. Can you update to the latest version and see if this fixes the problem?

**Comment by mpeterson2** [comment]
Closing old issues that are probably irrelevant to the code just to clear up the clutter. Go ahead and open a new issue if there is still a problem.
